wowup-electron/src/assets/i18n/de.json

```diff
@@ -115,55 +115,55 @@
+        "INSTALL_FAILED": "Beim Versuch das Addon zu installieren ist etwas schief gelaufen, bitte versuche es erneut.\n\nWenn diese Nachricht weiterhin erscheint, kannst Du auf unserem Discord im Kanal #wowup-support nach Hilfe fragen.",
         "INVALID_URL": "Die eingetragene URL ist ungültig. Beispiele für gültige URLs sind:\n\t- https://github.com/teelolws/Altoholic-Retail\n\t- https://www.wowinterface.com/downloads/info25610-8.3-014.html\n\t- https://www.curseforge.com/wow/addons/altoholic",
-        "NO_ADDON_FOUND": "Es wurde kein Addon gefunden. Stelle sicher, dass die URL zur richtigen Website führt.\n\nSolltest du versuchen ein Addon von GitHub zu installieren, stelle sicher, dass das Repository Releases Tags mit einem ZIP Archiv, welches das Addon enthält, besitzt.",
+        "NO_ADDON_FOUND": "Es wurde kein Addon gefunden. Stelle sicher, dass die URL zur richtigen Website führt.\n\nSolltest Du versuchen ein Addon von GitHub zu installieren, stelle sicher, dass das Repository ein Release Etikett mit einem ZIP Archiv hat, welches das Addon enthält.",
         "TITLE": "Addon Installation fehlgeschlagen"
       },
       "IMPORT_BUTTON": "Importieren",
       "INSTALL_BUTTON": "Installieren",
       "INSTALL_SUCCESS_LABEL": "Installiert!",
-      "SUPPORTED_SOURCES": "Unterstützt wird WowInterface und GitHub",
+      "SUPPORTED_SOURCES": "Unterstützt WowInterface und GitHub",
       "TITLE": "Installation eines Addons über eine externe URL"
     },
     "TELEMETRY": {
-      "DESCRIPTION": "Hilf mir, WowUp zu verbessern, indem du anonyme Installationsdateien und/oder Fehler schickst?",
-      "NEGATIVE_BUTTON": "Nein Danke",
-      "POSITIVE_BUTTON": "Sicher!",
+      "DESCRIPTION": "Möchest Du helfen, WowUp zu verbessern, indem anonyme Installationsdaten und/oder Fehler gesendet werden?",
+      "NEGATIVE_BUTTON": "Nein, Danke",
+      "POSITIVE_BUTTON": "Ja",
       "TITLE": "WowUp Telemetrie"
     }
   },
   "PAGES": {
     "ABOUT": {
-      "CHANGE_LOG_SECTION_LABEL": "Changelog / Änderungen",
+      "CHANGE_LOG_SECTION_LABEL": "Änderungsverlauf",
       "TITLE": "WowUp.io",
-      "WEBSITE_LINK_LABEL": "Schau Dir die Webseite an!"
+      "WEBSITE_LINK_LABEL": "Schau dir die Website an!"
     },
     "GET_ADDONS": {
       "CLIENT_TYPE_SELECT_LABEL": "World of Warcraft",
-      "INSTALL_FROM_URL_BUTTON": "Von URL installieren",
+      "INSTALL_FROM_URL_BUTTON": "Über eine URL installieren",
       "REFRESH_BUTTON": "Aktualisieren",
       "SEARCH_LABEL": "Suchen",
       "TABLE": {
         "ADDON_COLUMN_HEADER": "Addon",
-        "AUTHOR_COLUMN_HEADER": "Autor",
+        "AUTHOR_COLUMN_HEADER": "Veröffentlicher",
         "DOWNLOAD_COUNT_COLUMN_HEADER": "Anzahl Downloads",
         "PROVIDER_COLUMN_HEADER": "Anbieter",
-        "RELEASED_AT_COLUMN_HEADER": "Erscheinungsdatum",
+        "RELEASED_AT_COLUMN_HEADER": "Veröffentlichung",
```

Comment:
better is "Aktualisierungsdatum" or "Aktualisiert am"
Veröffentlichung is wrong, that is the date of the very first release of that add-on, not the update date